tests/unit/plugin/test_plugins.py

```
@@ -0,0 +1,1256 @@
+        self, empty_registry: _PluginRegistry, tmp_path: Path
+    ) -> None:
+        """load_manifest() handles empty YAML gracefully."""
+        manifest = write_manifest(tmp_path, "")
+        empty_registry.load_manifest(manifest)  # Should not raise
+
+    def test_load_manifest_invalid_schema_raises(
+        self, empty_registry: _PluginRegistry, tmp_path: Path
+    ) -> None:
+        """load_manifest() raises for invalid schema."""
+        manifest = write_manifest(tmp_path, "schema_version: [invalid, list]")
+        with pytest.raises(ValueError, match="Invalid plugins.yaml schema"):
```

Ruff (0.14.14)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)